client/rest-high-level/src/main/java/org/elasticsearch/client/TasksRequestConverters.java

```diff
@@ -54,4 +55,12 @@ static Request listTasks(ListTasksRequest listTaskRequest) {
             .putParam("group_by", "none");
         return request;
     }
+
+    static Request getTask(GetTaskRequest getTaskRequest) {
+        Request request = new Request(HttpGet.METHOD_NAME, "_tasks/" + getTaskRequest.getNodeId() + ":" + getTaskRequest.getTaskId());
+        RequestConverters.Params params = new RequestConverters.Params(request);
+        params.withTimeout(getTaskRequest.getTimeout()).withWaitForCompletion(getTaskRequest.getWaitForCompletion());
```

Comment:
can u put these on 2 lines? u can keep builder if u want, its just easier to see it split on 2 lines